rpc-jobs/RPC-AIO.yml

```diff
@@ -105,48 +105,40 @@
     dsl: |
       // CIT Slave node
       node() {{
-        try {{
-          dir("rpc-gating") {{
-              git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
-              common = load 'pipeline-steps/common.groovy'
-              pubCloudSlave = load 'pipeline-steps/pubcloud.groovy'
-              aio_prepare = load 'pipeline-steps/aio_prepare.groovy'
-              deploy = load 'pipeline-steps/deploy.groovy'
-              tempest = load 'pipeline-steps/tempest.groovy'
-              holland = load 'pipeline-steps/holland.groovy'
-          }}
-          instance_name = common.gen_instance_name()
-          pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
-          node(instance_name) {{
-            try {{
-              environment_vars = [
-                "DEPLOY_HAPROXY=yes",
-                "DEPLOY_AIO=no",
-                "DEPLOY_MAAS=no",
-                "DEPLOY_TEMPEST=no",
-                "DEPLOY_CEPH=${{DEPLOY_CEPH}}",
-                "DEPLOY_ELK=${{DEPLOY_ELK}}",
-                ]
-              aio_prepare.prepare()
-              deploy.deploy_sh(environment_vars: environment_vars)
-              deploy.addChecksumRule()
-              tempest.tempest()
-              holland.holland()
-              deploy.upgrade(environment_vars: environment_vars)
-              deploy.addChecksumRule()
-              tempest.tempest()
-            }} catch (e) {{
-              print(e)
-              throw e
-            }} finally {{
-              common.archive_artifacts()
-            }}
-          }} // pub cloud node
-        }} catch (e) {{
-          currentBuild.result = 'FAILURE'
```

Comment:
From chat:

```
@mattt no, because there is no top level exception handler - the exception will bubble up to jenkins itself and fail the build.
```